shaded-ning19/pom.xml

```diff
@@ -23,13 +23,14 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.apache.apex</groupId>
-    <artifactId>apex</artifactId>
-    <version>3.3.0-incubating</version>
+    <groupId>org.apache</groupId>
+    <artifactId>apache</artifactId>
+    <version>18</version>
+    <relativePath/>
   </parent>
   <groupId>org.apache.apex</groupId>
   <artifactId>apex-shaded-ning19</artifactId>
-  <version>1.0.0</version>
+  <version>1.1.0-SNAPSHOT</version>
```

Comment:
I will change the version to 1.0 instead of -SNAPSHOT so that it remains a "release" archive. There is no change compared to -incubating and no need to publish the artifact either.